e2e/tests/Login/SuccessfullyLogIn.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test } from '@playwright/test';
+
+import {
+  getAccessToken,
+  resetDB,
+} from '../../../services/API-service/test/helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  accessToken = await getAccessToken();
```

Comment:
It seems duplicate that you first log in already via API here, and then again via portal. In fact, resetting DB is not needed for this particular front-end login test.